http/webserver.go

```diff
@@ -239,15 +244,14 @@ func HandleStatic(h *config.WebStatic, w http.ResponseWriter, r *http.Request) (
 			}
 			http.Error(w, "403 - permission denied", http.StatusForbidden)
 			return true
-		} else if err == nil {
-			defer index.Close()
-			var ifi os.FileInfo
-			ifi, err = index.Stat()
-			if err == nil {
-				w.Header().Set("Content-Type", "text/html; charset=utf-8")
-				serveFile("index.html", ifi.ModTime(), index)
-				return true
-			}
+		}
+		defer index.Close()
```

Comment:
i can see index being nil with this change. err != nil is now still an option when we get here.